cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from cinema.views import movie_list, movie_detail
+
+app_name = "cinema"
+
+urlpatterns = [
+    path("movies/", movie_list, name="bus_list"),
+    path("movies/<int:pk>/", movie_detail, name="bus_detail"),
```

Comment:
try to use something like: name="movies-list" and name="movie-detail" instead of name="bus_list", name="bus_detail".

Reply:
Sorry🤣

Reply:
Don't worry it is minor error. I do that all time :)